### PR TITLE
Update create-edit-or-delete-a-custom-user-view.md

### PR DESCRIPTION
**Pull request by msbemba**
Added note on scenarios for unlicensed users with exchange mailboxes

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1348

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1348

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md [hunk 1]
```diff
@@ -84,6 +84,13 @@ You can also filter by additional user profile details used in your organization
 - **Unlicensed users** Select this box to find all the users who haven't been assigned a license. The results for this view can also include users who have an Exchange mailbox but don't have a license. To track those users specifically, use the filter **Unlicensed users with Exchange mailboxes or archives**. The results for this view can also include users who have an Exchange archive, but don't have a license.
     
 - **Unlicensed users with Exchange mailboxes or archives** Select this box to show user accounts that were created in Exchange Online and have an Exchange mailbox, but weren't assigned an Office 365 license. The results of this filter include users who have or who were assigned an Exchange archive. 
+
+> [!NOTE]
+> The unlicnesed users with exchange mailboxes works under the following situaions
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md]:
```suggestion
> The **Unlicensed users with Exchange mailboxes** filter works when:
```

File: microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md [hunk 2]
```diff
@@ -84,6 +84,13 @@ You can also filter by additional user profile details used in your organization
...
+1. The mailbox has been recently converted from shared to user and it has no license.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md]:
```suggestion
1. The mailbox has been recently converted from **shared** to **user** and it has no license.
```

File: microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md [hunk 3]
```diff
@@ -84,6 +84,13 @@ You can also filter by additional user profile details used in your organization
...
+2. The mailbox has been recently migrated to O365 but license has not been assigned.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md]:
```suggestion
2. The mailbox has been recently migrated to Office 365 but a license has not been assigned.
```

File: microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md [hunk 4]
```diff
@@ -84,6 +84,13 @@ You can also filter by additional user profile details used in your organization
...
+3. The mailbox has been created using powershell, and a license has not been assigned.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md]:
```suggestion
3. The mailbox has been created using PowerShell, and a license has not been assigned.
```

File: microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md [hunk 5]
```diff
@@ -84,6 +84,13 @@ You can also filter by additional user profile details used in your organization
...
+4. A new mailbox is provisioned for user created on-premise with New-RemoteMailbox cmdlet.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md]:
```suggestion
4. A new mailbox that has been created on-premise with a New-RemoteMailbox cmdlet is provisioned for the user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@msbemba  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@kwekuako - PR has been copy edited and is ready for final review. This is based on issue #1348.

Thank you.

cc: @kenwith

**Pull request merged**